### DISCUSSION TITLE
I think Software Manager should not show results that are not apps

### DISCUSSION THREAD

**Discussion by marco4mp** [linuxmint/discussions#476] [Ideas]
![Software Manager KDE](https://github.com/linuxmint/discussions/assets/52709674/adaf821c-ea55-4e97-bad4-d2eb73179483)

It does not seem to me that it is safe for general users to be able to install (or uninstall) software that can cause profound changes to the system.

Easily from the Software Manager you can remove important packages from the system or you can also install the entire KDE ecosystem,  generating the typical problems that we can see in different forums.

**Comment by Secret-chest** [comment]
Why is installing KDE a problem?

**Reply by Jopp-gh** [reply]
I think people like to mix and match, but when we install system relevant data, we can't install everything random we like, but only matching software parts. 
Kde is kde, gnome is gnome, etc. I think one excludes the other, to prevent problems and this assumption seems logic to me at least. So I subscribe the Op's concern.
